.github/workflows/vib-thanos.yaml

```
@@ -0,0 +1,41 @@
+name: 'vib'
+on: # rebuild any PRs and main branch changes
+  pull_request_target:
+    branches:
+      - master
+    paths:
+      - 'bitnami/thanos/*'
+env:
+  CSP_API_URL: https://console.cloud.vmware.com
+  CSP_API_TOKEN: ${{ secrets.CSP_API_TOKEN }}
+  VIB_PUBLIC_URL: https://cp.bromelia.vmware.com
+jobs:
+  vib-bucketweb:
```

```suggestion
  vib-verify-bucketweb:
```
In order to follow the same convention for multiple jobs, as in `vib-redis.yaml`

---

Great catch! Fixed it!